src/algebra/homology/additive.lean

```diff
@@ -52,6 +52,12 @@ function.injective.add_comm_group hom.f
 
 instance : preadditive (homological_complex V c) := {}
 
+/-- The `i`-th component of a chain map, as an additive map from chain maps to morphisms. -/
+@[simps]
+def hom.f_add_monoid_hom {C₁ C₂ : homological_complex V c} (i : ι) :
```

Comment:
I'm not sure dot notation will actually work with this for `F : C₁ ⟶ C₂`, so perhaps we should shorten the name a bit?

Reply:
I was going to findability over typability, using the logic that this is a version of `homological_complex.hom.f` and so should be named by adding a suffix to that. I think just `f_hom` is insufficient, as we'll also have the linear map version soon enough.